TODO.md

```diff
@@ -0,0 +1,17 @@
+
+
+# Temporary file, to log todos
+
+
+one pr for multiple image changes (?) need to handle this case later on
+add retry mechanism, if task fails after triggering the pr_script
+use GITHUB_TOKEN instead of GITHUB_USER and GITHUB_PASSWORD, for Hub
+test with real repo push (to test credentials)
```

Comment:
Not tested on real repo yet?

Reply:
No, will be tested with the real in few days. 
The only (expected!) difference will be that with the real repo, the PR will be opened for real, not just a commit creation.